src/dupsfindr.go

```diff
@@ -17,97 +18,97 @@ func main() {
 	directory := flag.String("directory", "", "the absolute directorypath to search for duplicates")
 	flag.Parse()
 
-	fmt.Println("DupsFindr started with: ", *directory)
+	if len(*directory) == 0 {
```

Comment:
Checking the length and then panicking increases readability and reduces indentations.